exonum-java-binding/core/src/test/java/com/exonum/binding/core/runtime/ServiceRuntimeAdapterTest.java

```diff
@@ -123,7 +126,7 @@ void addService() throws CloseFailuresException {
 
     // Check the runtime was invoked with correct config
     ServiceInstanceSpec expected = ServiceInstanceSpec.newInstance(serviceName, serviceId,
-        ServiceArtifactId.newJavaId(javaArtifactName));
+        ServiceArtifactId.newJavaId(artifactId));
     verify(serviceRuntime).startAddingService(fork, expected, configuration);
```

Comment:
I can't understand why this test isn't working.
```
[ERROR] Failures: 
[ERROR]   ServiceRuntimeAdapterTest.addService:130 
Argument(s) are different! Wanted:
serviceRuntime.startAddingService(
    com.exonum.binding.core.storage.database.Fork@76c12d09,
    ServiceInstanceSpec{name=s1, id=1, artifactId=1:com.acme:foo:1.2.3},
    [(byte) 0x01, (byte) 0x02]
);
-> at com.exonum.binding.core.runtime.ServiceRuntime.startAddingService(ServiceRuntime.java:189)
Actual invocations have different arguments:
serviceRuntime.startAddingService(
    com.exonum.binding.core.storage.database.Fork@76c12d09,
    ServiceInstanceSpec{name=s1, id=1, artifactId=1:com.acme:foo:1.2.3},
    [(byte) 0x01, (byte) 0x02]
);
-> at com.exonum.binding.core.runtime.ServiceRuntimeAdapter.startAddingService(ServiceRuntimeAdapter.java:131)
```

Reply:
Their _string representation_ is identical; not the objects (artifactId most certainly — I suspect one has name 'com.acme' + version 'foo:1.2.3', the other — normal).